libcxx/include/istream

```diff
@@ -291,6 +293,13 @@ public:
   basic_istream& getline(char_type* __s, streamsize __n, char_type __dlm);
 
   basic_istream& ignore(streamsize __n = 1, int_type __dlm = traits_type::eof());
+#    if _LIBCPP_STD_VER >= 26
+  _LIBCPP_HIDE_FROM_ABI basic_istream& ignore(streamsize __n, char_type __delim)
```

Comment:
```suggestion
  basic_istream& ignore(streamsize __n, char_type __delim)
```

Reply:
I _guess_ we can make this a function template to work around the restrictions with DLL. Yeah, this will also reduce the ambiguity in the paper, but I think such an approach is not a bad thing, and still conforming per [[member.functions]/2](https://wg21.link/member.functions#2).